.github/workflows/ci-config.yml

```diff
@@ -0,0 +1,40 @@
+# Workflow for processing and publishing bikeshed proposals to github pages
+
+name: verify proposals
+on:
+  push:
+    # only concerned with changes in the main branch
+    branches: [ main ]
```

Comment:
Does the verify need to run on push as well?

Reply:
Probably not. That happens with the CD workflow. For the purposes of this PR, I really wanted to just copy what currently exists. We can modify/refine in subsequent PRs.